### PR TITLE
Adding Notifications Project to add email functionality

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

Made new shared notifications project and test project to send emails with smtp server

## Why?

NAC-769

## This PR has:

- [X] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [X] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

**Review by bgfoshay — CHANGES_REQUESTED** [review]
Looks like a good start for notification support! Couple changes we still need though- you should additionally add the new projects to the shared.sln solution file. Also, the tests should be added to the circleci coverage and we need dependabot to cover these solutions as well.

File: system.sln [hunk 1]
```diff
@@ -152,6 +152,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Piipan.States.Core.Integrat
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Piipan.States.Core.Tests", "states\tests\Piipan.States.Core.Tests\Piipan.States.Core.Tests.csproj", "{FA44CF06-EC0C-422F-8A39-8B222DDAAD00}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Piipan.Shared.Notifications", "shared\src\Piipan.Shared.Notifications\Piipan.Shared.Notifications.csproj", "{04D97589-4A30-4225-AE0F-455E53AE6337}"
```
Comment (ccaprio97) [inline system.sln:155]: @bgfoshay  the new projects are in the system solution. Do you mean something else?

Comment (bgfoshay) [reply system.sln:155]: In the shared folder, there is a shared.sln file. We should make sure to add the projects there as well.

**Review by bgfoshay — APPROVED** [review]
👍

**Pull request merged**